index.js

```diff
@@ -2746,6 +2748,59 @@
         return filter(is(type), Just(x));
       });
 
+  //# keys :: StrMap a -> Array String
+  //.
+  //. Returns an array of the keys of the supplied StrMap.
```

Comment:
We could omit "an array of" as this is indicated by the type signature.

Let's append "in arbitrary order".